rfc/README.md

```diff
@@ -0,0 +1,26 @@
+# Request for Comments (RFC)
+
+This directory hosts proposals (RFCs) of evolutionary topics around VAST. The
+objective is to actively include the community in the discussion.
+
+## Add a new RFC
+
+The workflow to add a new RFC works as follows:
+
+1. Copy the template and increase the running counter:
+   ```bash
+   # Assumption: the last proposal is 041-proposal.md
+   cp 000-TEMPLATE.md 042-proposal-name.md
+   ```
+
+2. Write the proposal in Github Markdown.
+
+3. [Submit a pull request](https://github.com/tenzir/vast/pulls).
+
+4. Share the pull request and drive the discussion forward.
```

Comment:
Maybe precise the announcement channels for the pull request? (would be useful to know as a commentator as well)

Reply:
I pointed out the Community Slack. Should we go broader? I thought about GitHub Discussions, but given that we have a PR already, I'm not sure if there is additional benefit.

Reply:
I think Slack and the github PR are great. For future RFC proposers and reviewers, I thought it may just be good to write this down.